core/domain/subtopic_page_domain.py

```diff
@@ -19,13 +19,13 @@
 import copy
 
 from constants import constants
-from core.domain import html_cleaner
+from core.domain import state_domain
 from core.platform import models
 import utils
 
 (topic_models,) = models.Registry.import_models([models.NAMES.topic])
 
-SUBTOPIC_PAGE_PROPERTY_HTML_DATA = 'html_data'
+SUBTOPIC_PAGE_PROPERTY_PAGE_CONTENTS = 'page_contents'
```

Comment:
I think we should create a new property? that will make a separate space/property on which translator can work!

Reply:
done